pkg/services/sqlstore/user.go

```diff
@@ -195,6 +196,27 @@ func (ss *SQLStore) CloneUserToServiceAccount(ctx context.Context, siUser *model
 		IsServiceAccount: true,
 	}
 
+	newuser, err := ss.CreateUser(ctx, cmd)
+	if err != nil {
+		ss.log.Warn("user not cloned", "err", err)
+		return nil, errors.Errorf("Failed to create user: %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to create user: %v", err)
```

`github.com/pkg/errors` has been folded into the standard library and Go typically doesn't start errors with an upper-case letter.